### PR TITLE
Upgrade Vuetify for AB#16537

### PR DESCRIPTION
**Pull request by BrianMaki**
# Fixes [AB#16537](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/16537)

## Description

- Upgraded Vuetify to version 3.5.1
- Fixed compilation errors

**Issues:**

- useDisplay() from App Store has to be called within a setup function, i.e. inside one of our component's <script setup> sections, not in main.ts.
- useDisplay() from App Store is currently used inside useAppStore(), which is called by useUserStore(), which is called by main.ts (and called by useAuthStore() and useNotificationStore(), also called by main.ts).

**Solution:**

- Move isMobile into a different store like navigation store.
- Renamed navigation store to layout store

## Testing

- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [x] Not Required

## UI Changes

No

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

File: Apps/WebClient/src/ClientApp/src/components/site/HeaderComponent.vue [hunk 1]
```diff
@@ -9,21 +9,20 @@ import RatingComponent from "@/components/site/RatingComponent.vue";
...
 const headerScrollThreshold = 100;
 
-const appStore = useAppStore();
+const layoutStore = useLayoutStore();
 const configStore = useConfigStore();
 const userStore = useUserStore();
 const notificationStore = useNotificationStore();
 const authStore = useAuthStore();
-const navigationStore = useNavigationStore();
+const navigationStore = useLayoutStore();
```
Comment (MikeLyttle) [inline Apps/WebClient/src/ClientApp/src/components/site/HeaderComponent.vue:25]: The `const navigationStore` line won't be needed if other references to `navigationStore` in the file are replaced with `layoutStore`.

File: Apps/WebClient/src/ClientApp/src/components/site/SidebarComponent.vue [hunk 2]
```diff
@@ -2,22 +2,21 @@
...
-import { useNavigationStore } from "@/stores/navigation";
+import { useLayoutStore } from "@/stores/layout";
 import { useUserStore } from "@/stores/user";
 
-const appStore = useAppStore();
+const layoutStore = useLayoutStore();
 const authStore = useAuthStore();
 const configStore = useConfigStore();
-const navigationStore = useNavigationStore();
+const navigationStore = useLayoutStore();
```
Comment (MikeLyttle) [inline Apps/WebClient/src/ClientApp/src/components/site/SidebarComponent.vue:13]: Same comment as previous.

File: Apps/WebClient/src/ClientApp/src/components/site/HeaderComponent.vue (hunk 1 above)
Comment (BrianMaki) [reply Apps/WebClient/src/ClientApp/src/components/site/HeaderComponent.vue]: Fixed

File: Apps/WebClient/src/ClientApp/src/components/site/SidebarComponent.vue (hunk 2 above)
Comment (BrianMaki) [reply Apps/WebClient/src/ClientApp/src/components/site/SidebarComponent.vue]: Fixed.

**Review by MikeLyttle — APPROVED** [review]

**Review by BronzBierd — APPROVED** [review]

**Pull request merged**